service-registry/registry-local/src/main/java/org/apache/servicecomb/localregistry/LocalRegistryStore.java

```diff
@@ -83,9 +90,28 @@ public void run() {
     selfMicroserviceInstance.setInstanceId(selfMicroservice.getServiceId());
     selfMicroserviceInstance.setServiceId(selfMicroservice.getServiceId());
 
-    InputStream is = this.getClass().getClassLoader().getResourceAsStream(REGISTRY_FILE_NAME);
-    if (is != null) {
-      initFromData(is);
+    InputStream is = null;
+
+    try {
+      ClassLoader loader = JvmUtils.findClassLoader();
```

Comment:
try org.apache.servicecomb.foundation.common.utils.ResourceUtil#findResourcesBySuffix?

Reply:
I see many ohter places code like this. 
'ResourceUtil#findResourcesBySuffix' returns List<URI> and not convinient for open stream.